samples/nextjs/src/components/Layout.tsx

```diff
@@ -72,7 +66,7 @@ const Layout = ({ route, sitecoreContext }: LayoutProps) => {
 
         VI detection only runs once for a given analytics ID, so this is not a recurring operation once cookies are established.
       */}
-      {!sitecoreContext.pageEditing && <VisitorIdentification />}
```

Comment:
@ambrauer You removed page editing check inside VisitorIdentification component, should you revert changes here?

Reply:
@sc-illiakovalenko No, this check was made originally in the POC as a quick way to get past this issue with the VisitorIdentification control. But no longer required now that it's resolved.